client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -387,6 +387,25 @@ export function getEnhancedTasks(
 						completed: isEmailVerified,
 					};
 					break;
+				case 'set_up_payments':
+					taskData = {
+						actionDispatch: () => {
+							recordTaskClickTracksEvent( flow, task.completed, task.id );
+							window.open( `/earn/payments/${ siteSlug }#launchpad`, '_blank' );
```

Comment:
Why `_blank`and not in the same tab?

Reply:
there's no clear way to get back to the Launchpad after going to the Earn page. Someone suggested new tab 🤔 , I think it makes sense for now.

Maybe a better pattern could be to make these links instead?

Reply:
That's valid concern, but I don't think this solves it. :-) Folks get lost with multiple tabs all of a sudden open, browser back button stops working, and it's tricky especially on mobile.

I think we need to look at the "My home Launchpad" work and think if "My home" navigation item in the sidebar could be changed somehow to solve this, because I agree it's a real problem.

Reply:
@simison so no newtab for now?

Reply:
Let's find a better way to solve the problem and remove `_blank`

I wrote about this in p2 (p9Jlb4-7c4-p2/#comment-7904) but maybe we can think of something intermediate.